### PR TITLE
Create time_conversions.py

### PR DESCRIPTION
**Pull request by AkshitGulyan**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: conversions/time_conversions.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+def convert_time(time, from_unit, to_unit):
+    units = {"s": 1, "min": 60, "h": 3600, "d": 86400}
+
+    if from_unit in units and to_unit in units:
+        conversion_factor = units[from_unit] / units[to_unit]
+        converted_time = time * conversion_factor
+        return converted_time
+    else:
+        return "Invalid units provided."
```
Comment (cclauss) [inline conversions/time_conversions.py:27]: Please `raise ValueError("Invalid units provided.")` instead.

File: conversions/time_conversions.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+"""
```
Comment (cclauss) [inline conversions/time_conversions.py:30]: Please move this docstring just below line 19.

File: conversions/time_conversions.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+def convert_time(time, from_unit, to_unit):
+    units = {"s": 1, "min": 60, "h": 3600, "d": 86400}
+
+    if from_unit in units and to_unit in units:
+        conversion_factor = units[from_unit] / units[to_unit]
+        converted_time = time * conversion_factor
+        return converted_time
```
Comment (CaedenPH) [inline conversions/time_conversions.py:25]:
```suggestion
        return time * conversion_factor
```
No need to have this variable

File: conversions/time_conversions.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+REFERENCES :
+
+https://en.wikipedia.org/wiki/Unit_of_time
+https://en.wikipedia.org/wiki/Time
+
+"""
+
+
+def convert_time(time, from_unit, to_unit):
+    units = {"s": 1, "min": 60, "h": 3600, "d": 86400}
```
Comment (amirsoroush) [inline conversions/time_conversions.py:20]: Since you used single character for "second", "hour" and "day", use `"m"` for "minute" to be consistent.

**Pull request closed without merging**